src/client/src/.tests/components/App.test.js

```diff
@@ -0,0 +1,16 @@
+const React = require('react')
+const {shallow, configure} = require('enzyme')
+const Adapter = require('enzyme-adapter-react-16')
+const App = require('../../App')
+
+const ONE_ELEMENT = 1
+configure({adapter: new Adapter()})
+
+describe('<App />', () => {
+	const wrapper = shallow(<App />)
+
+	it('renders without crashing', () => {
+		expect(wrapper.render().length).toBe(ONE_ELEMENT)
```

Comment:
What I meant by magic numbers, is to give them meaningul name. I.e.
```
const EXPECTED_ELEMENTS_COUNT = 1
const elementsCount = wrapper.render().length
expect(elementsCount).toBe(EXPECTED_ELEMENTS_COUNT)
```

Reply:
kk